public/planning/poste/class.planning.php

```diff
@@ -415,10 +420,23 @@ public function notifications()
             $agent = $agents[$id];
             $start = $elem['date'] . ' ' . $elem["debut"];
             $end = $elem['date'] . ' ' . $elem["fin"];
-            if ($agent->isAbsentOn($start, $end)) {
-                $elem["absent"] = 1;
+
+            // Looking for absences.
+            $filter = '';
+            $position = isset($postes[$elem['poste']]) ? $postes[$elem['poste']] : null;
+            if ($position && $position['teleworking'] == 1) {
+                $teleworking_exception = (!empty($teleworking_reasons))
+                    ? "AND `motif` NOT IN ('" . implode("','", $teleworking_reasons) . "')"
+                    : null;
+
+                $filter .= " $teleworking_exception";
             }
 
+            $db = new \db();
+            $db->select('absences', '*', "`debut`<'$end' AND `fin`>'$start' AND `perso_id`='$id' $filter ");
```

Comment:
TESTS : Les absences non-validées sont prises en compte alors qu'elles en devraient pas l'être.
Il faut ajouter AND valide  > 0